napari_btrack/track.py

```diff
@@ -305,15 +305,13 @@ def _widgets_to_tracker_config(container: Container) -> TrackerConfig:
     for default_name, default_value in zip(
         ["measurements", "states", "dt", "prob_not_assign", "name"],
         [3, 6, 1.0, 0.001, f"{mode}_motion"],
-        strict=True,
```

Comment:
Oh this is such an unsatisfying fix 😢

Reply:
I think it's the only option unless we only support Python >= 3.10 😞

Reply:
or we [implement the check ourselves](https://stackoverflow.com/a/69485272)... but that seems a little extreme - it's pretty easy to see here that each list has five elements

Reply:
Yeah there's no point. I've ignored it with a comment in the config for now